ci/travis/ci.sh

```diff
@@ -111,15 +136,21 @@ build() {
 
     # This command should be kept in sync with ray/python/README-building-wheels.md,
     # except the "${MOUNT_BAZEL_CACHE[@]}" part.
-    "${WORKSPACE_DIR}"/ci/suppress_output docker run --rm -w /ray -v "${PWD}":/ray "${MOUNT_BAZEL_CACHE[@]}" -e TRAVIS_COMMIT="${TRAVIS_COMMIT}" -ti rayproject/arrow_linux_x86_64_base:python-3.8.0 /ray/python/build-wheel-manylinux1.sh
+    suppress_output docker run --rm -w /ray -v "${PWD}":/ray "${MOUNT_BAZEL_CACHE[@]}" -e TRAVIS_COMMIT="${TRAVIS_COMMIT}" -ti rayproject/arrow_linux_x86_64_base:python-3.8.0 /ray/python/build-wheel-manylinux1.sh
   fi
 
   if [ "${MAC_WHEELS-}" = 1 ]; then
     # This command should be kept in sync with ray/python/README-building-wheels.md.
-    "${WORKSPACE_DIR}"/ci/suppress_output "${WORKSPACE_DIR}"/python/build-wheel-macos.sh
+    suppress_output "${WORKSPACE_DIR}"/python/build-wheel-macos.sh
+  fi
+}
+
+_main() {
```

Comment:
why do we need this extra _main?

Reply:
I'll need it in the next PR since it where I'll call `reload_env`, but I can take it out for now.

BTW I replied to your other comments, but somehow they're not showing up under your review for me. You can click on the other threads to see them.